### ISSUE TITLE
Svelte 4 support?

### ISSUE THREAD

**Issue by homostellaris** [Oli8/spaper#87]
Thanks for the library, I'm using it at https://scheduleyour.social and its working great 🙏 

I'm going to try upgrading to Svelte 4 and check everything still works, my understanding is that most libraries won't be broken by upgrading to Svelte 4 but just wanna check if there are any known reasons why this definitely won't work before I invest too much time? And if any changes _are_ required, will you accept a PR?

**Comment by Oli8** [comment]
Thanks for the kind words and the PR, I'll try to check this out over the week-end

**Comment by Oli8** [comment]
Tested the current version of spaper on a v4 Svelte (with rollup) app.

Those components are broken:
- Tabs
- Carousel

Those are affected:
- Collapsible
- Slider
- Pagination
- Popover
- Toast

**Comment by homostellaris** [comment]
> Tested the current version of spaper on a v4 Svelte (with rollup) app.
> 
> Those components are broken:
> - Tabs
> - Carousel
> 
> Those are affected:
> - Collapsible
> - Slider
> - Pagination
> - Popover
> - Toast
> 

Ah that's a shame. Are you intending to fix them? If you share your testing methodology I could help?

**Comment by homostellaris** [comment]
Just FYI I've been using this in earnest with Svelte 4 and I've not had any issues. My app only uses Popover and Toast though.

**Comment by shiftlabs1** [comment]
@Oli8 , I think the ask should be plans to upgrade to svelte5 (or at least a branch version for svelte 5) using runes

**Comment by Oli8** [comment]
> @Oli8 , I think the ask should be plans to upgrade to svelte5 (or at least a branch version for svelte 5) using runes

Yep, agreed, I'll get to it eventually but can't know when, contributions are welcome
